README.md

```diff
@@ -658,6 +659,30 @@ The ids passed will be appended to the list of already accepted vendors, categor
 
 It's important to notice, this method is intended to be used for **custom** vendors and purposes only. For IAB vendors and purposes, it's still required to get consents via the consent message or privacy manager.
 
+## Programmatically delete the saved consent
+Using the same strategy for the custom consent, it's possible to programmatically delete the current user consent to a list of vendors, categories and legitimate interest categories by using the following method from the consent lib:
+Kotlin
+```kotlin
+            spConsentLib.deleteCustomConsentTo(
+                vendors = listOf("5ff4d000a228633ac048be41"),
+                categories = listOf("608bad95d08d3112188e0e36", "608bad95d08d3112188e0e2f"),
+                legIntCategories = emptyList(),
+                success = { spCustomConsents -> println("custom consent: [$spCustomConsents]") }
+            )
+```
+Java
+```java
+            spConsentLib.deleteCustomConsentTo(
+                    Arrays.asList("5ff4d000a228633ac048be41"),
+                    Arrays.asList("608bad95d08d3112188e0e36", "608bad95d08d3112188e0e2f"),
+                    new ArrayList<>(),
+                    (SPConsents) -> {  return Unit.INSTANCE;  }
+            )
+```
+The ids passed will be appended to the list of already accepted vendors, categories and leg. int. categories. The method is asynchronous so you must pass a `Runnable` that will receive back an instance of `GDPRUserConsent` in case of success or it'll call the `onError` callback in case of failure.
```

Comment:
Correction:

The ids passed will be ~appended~ removed to the list of already...